# 9DジャイロセンサGET
---
float64 gyrX
float64 gyrY
float64 gyrZ

